packages/admin-panel/src/routes/entities/entities.js

```diff
@@ -109,13 +92,20 @@ const IMPORT_CONFIG = {
   ],
 };
 
-export const EntitiesPage = props => (
-  <ResourcePage
-    title="Entities"
-    endpoint={ENTITIES_ENDPOINT}
-    columns={COLUMNS}
-    expansionTabs={EXPANSION_CONFIG}
-    importConfig={IMPORT_CONFIG}
-    {...props}
-  />
-);
+export const entities = {
+  title: 'Entities',
+  path: '',
+  default: true,
+  endpoint: ENTITIES_ENDPOINT,
+  columns: COLUMNS,
+  importConfig: IMPORT_CONFIG,
+  needsBESAdminAccess: ['delete'],
+  detailsView: {
+    title: 'Survey Responses',
+    endpoint: 'entities/{id}/surveyResponses',
```

Comment:
`ENTITIES_ENDPOINT` is defined up top, though I’m not sure if this is actually an improvement?

```suggestion
    endpoint: `${ENTITIES_ENDPOINT}/{id}/surveyResponses`,
```